### DISCUSSION TITLE
Konfigurowalny interwal pobierania danych

### DISCUSSION THREAD

**Discussion by Nonekk** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#160] [General]
Witam, czy jest szansa na dodanie w konfiguracji pola do zmiany interwalu pobierania danych z elicznika ?

Pozdrawiam

**Comment by PiotrMachowski** [comment]
Potencjalnie jest. Ale po co?

**Reply by Nonekk** [reply]
Teoretycznie po 8.5h przerwy powinno odblokowac w razie blokady antyspamowej, ale czesto to sie nie dzieje u mnie i wtedy wylaczam integracje na noc i rano zalaczam i dziala wszystko poprawnie. Mozliwe ze ktos z aplikacji momentami podbija czas potrzebny do oblokowania logowania. Odswiezanie danych co 12 lub 24h pewnie by mi tez wystarczylo.

**Reply by PiotrMachowski** [reply]
Na razie możesz manualnie zmienić interwał tutaj: https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/blob/master/custom_components/tauron_amiplus/const.py#L55C1-L55C24
